cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md

```diff
@@ -27,29 +26,27 @@ Digest: sha256:6b9a12992ba088a168b87df98a841d3c56dede326684f5551368fd359acfb43c
 Status: Downloaded newer image for clelange/cmssw:5_3_32
 Setting up CMSSW_5_3_32
 CMSSW should now be available.
-[15:10:10] cmsusr@eb9ecf54fd2a ~/CMSSW_5_3_32/src $
 ```
 
-Once done, you should see the commmand prompt (`docker run -i` means run interactively):
+Once done, you should see the commmand prompt for the CMSSW instance within Docker:
 
-`
+```console
 cmsusr@eb9ecf54fd2a ~/CMSSW_5_3_32/src $
-`
+```
 
 ### Instructions for 2010 data
 
 Once Docker is installed, you can fetch a CMSSW image and create and start a container using the `docker run` command:
 
-`
-$ docker run --name opendata-2010 -it clelange/cmssw:4_2_8 /bin/bash
-`
+```sh
+docker run --name opendata-2010 -it clelange/cmssw:4_2_8 /bin/bash
+````
```

Comment:
Should be three backticks only

Reply:
Otherwise looks good. I'll fix this during rebase and merge

Reply:
Ack! Sorry! Thanks for spotting and for fixing. :)